ChangeLog

```diff
@@ -10,6 +10,11 @@ Features
    * Added the macro MBEDTLS_X509_MAX_FILE_PATH_LEN that enables the user to
      configure the maximum length of a file path that can be buffered when
      calling mbedtls_x509_crt_parse_path().
+   * Added a configuration file config-no-entropy.h that enables a subset of
+     library features that do not require an entropy source.
+   * Added the macro MBEDTLS_ENTROPY_MIN_HARDWARE in config.h. This allows users
+     to configure the minimum number of bytes for their specific implementations
+     of hardware entropy sources.
```

Comment:
I'd phrase this as 'allows users to configure the minimum number of bytes for entropy sources using the 'mbedtls_hardware_poll()' function.' Just to be more specific.